src/main/java/gmail/Lance5057/items/HeaterShield.java

```diff
@@ -220,18 +220,18 @@ public void addInformation(ItemStack par1ItemStack, EntityPlayer par2EntityPlaye
         super.addInformation(par1ItemStack, par2EntityPlayer, par3List, par4);
         par3List.add("");
         par3List.add(
-                EnumChatFormatting.DARK_GREEN
-                        + ItemStack.field_111284_a.format(
-                                1F / (10f / (tags.getCompoundTag("InfiTool").getInteger("MiningSpeed") / 1.5f)) / 20F)
-                        + StatCollector.translateToLocal("attribute.shield.block.time"));
+            EnumChatFormatting.DARK_GREEN
+                + String.format(
+                    StatCollector.translateToLocal("attribute.shield.block.time"),
+                    ItemStack.field_111284_a.format(
+                        1F / (10f / (tags.getCompoundTag("InfiTool").getInteger("MiningSpeed") / 1.5f)) / 20F)));
```

Comment:
Instead of String.format(), this could be simplified slightly by using StatCollector.translateToLocalFormatted()